app/javascript/lib/vue-components/modules/Table.vue

```diff
@@ -0,0 +1,232 @@
+            :class="{ 'is-clickable': rowClickable }"
+            class="gobierto-table__tr"
+            @click="onRowClick(item)"
+          >
+            <template v-for="[id, { name, index, type, cssClass }] in arrayColumnsFiltered">
+              <template v-if="type === 'money'">
+                <td
+                  :key="id"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>
+                    {{ item[id] | money }}
+                  </span>
+                </td>
+              </template>
+              <template v-else-if="type === 'date'">
+                <td
+                  :key="id"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>
+                    {{ item[id] | date }}
+                  </span>
+                </td>
+              </template>
+              <template v-else-if="type === 'truncate'">
+                <td
+                  :key="id"
+                  class="gobierto-table__td"
+                >
+                  <span :class="cssClass">
+                    {{ item[id] }}
+                  </span>
+                </td>
+              </template>
+              <template v-else>
+                <td
+                  :key="id"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>{{ item[id] }}</span>
+                </td>
+              </template>
+            </template>
+          </tr>
+        </template>
+      </tbody>
+    </table>
+    <template v-if="showPagination">
+      <slot
+        name="pagination"
+        :show-data="updateData"
+      >
+        <Pagination
+          :data="rowsSorted"
+          :items-per-page="25"
+          :container-pagination="'main'"
+          @showData="updateData"
+        />
+      </slot>
+    </template>
+  </div>
+</template>
+<script>
+import { Pagination } from "lib/vue-components";
+import TableColumnsSelector from './TableColumnsSelector'
+import SortIcon from './SortIcon'
+import { VueFiltersMixin } from "lib/vue/filters"
+export default {
+  name: 'Table',
+  components: {
+    SortIcon,
+    TableColumnsSelector,
+    Pagination,
+  },
+  mixins: [VueFiltersMixin],
+  defaults: {
+    sortColumn: "id",
+    sortDirection: "up",
+  },
+  props: {
+    data: {
+      type: Array,
+      default: () => []
+    },
+    columns: {
+      type: Array,
+      default: () => []
+    },
+    orderColumn: {
+      type: String,
+      default: ''
+    },
+    showColumns: {
+      type: Array,
+      default: () => []
+    },
+    showColumnSelector: {
+      type: Boolean,
+      default: true
+    },
+    showPagination: {
+      type: Boolean,
+      default: true
+    },
+    rowClickable: {
+      type: Boolean,
+      default: true
+    },
+    onRowClick: {
+      type: Function,
+      default: x => x
+    }
```

Comment:
It's redundant declaring `onRowClick` as a function, and then, add another prop `rowClickable` to set the styles.

A row should be clickable (cursor), when `onRowClick` has any value. So, first off, remove the `rowClickable` prop. Then, set `default: null` for onRowClick. And lastly, create a computed prop who checks whether onRowClick is defined or not. If so, you may something like:

```vue
<tr
  :key="index"
  :class="{ 'is-clickable': isRowClickable }"
  class="gobierto-table__tr"
  @click="isRowClickable ? onRowClick(item) : null"
>
```
```vue
computed: {
  isRowClickable() {
    return !!this.onRowClick
  }
}
```